src/main/java/org/sasanlabs/internal/utility/FileReader.java

```diff
@@ -0,0 +1,29 @@
+package org.sasanlabs.internal.utility;
+
+import org.sasanlabs.service.vulnerability.fileupload.UnrestrictedFileUpload;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.InputStream;
+
+
+public class FileReader {
```

Comment:
I think you have created this class such that you can mock the `readAsStream` functionality right?
I think you don't need a different class, create a `spy` for  PreflightController and a method in PreflightController which you can mock using spy and everything else will work perfectly.

Reply:
Yes, that was the purpose for the class. 
Ok. I will make the changes.